### PR TITLE
DO NOTE MERGE: update docs, particularly related to configuration

### PR DESCRIPTION
**Pull request by rgitzel**
I'm creating this PR early to start getting feedback and corrections.

### CODE REVIEW

File: docs/source/basic_concepts.rst [hunk 1]
```diff
@@ -1,41 +1,70 @@
...
+  others, even if deployed in the same region and account
+- Corresponds to a Serverless stage, a Terraform workspace, etc.
+- typical names would be "dev", "qa" and "prod"
+
+  * can also be for specific people ("dev-alice", "dev-bob"), tickets ("qa-234") or for any other reason
+
+- Each environment will have its own unique set of configuration values
+
+
+THIS NEEDS TO GO ELSEWHERE
```
Comment (rgitzel) [inline docs/source/basic_concepts.rst]: Some of this is reproduced elsewhere, or should be put elsewhere...

File: docs/source/runway_config.rst [hunk 2]
```diff
@@ -3,6 +3,251 @@
...
+
+Within a specific module, the associated deployment tool (e.g. Serverless or Terraform) will manage the order
+that resources are created (or deleted), ensuring that dependencies are created earlier (or deleted later).
+Resources might even be created or deleted in parallel, if the deployment tool determines there are no dependencies
+between them.
+
+Within a deployment, however, modules will be processed in the order specified in ``runway.yml``; so in this example,
+the Stacker module ``database.cfn`` will be deployed before the Serverless module ``myapp.sls``.  Or, when
+destroying the deployment, ``myapp.sls`` will be destroyed before ``database.cfn``.  Runway does not
+attempt to determine the order itself based on dependencies between the modules.  **((VERIFY!!))**
```
Comment (rgitzel) [inline docs/source/runway_config.rst]: True?

File: docs/source/runway_config.rst [hunk 3]
```diff
@@ -3,6 +3,251 @@
...
+          - us-east-1
+          - us-east-2
+          - us-west-1
+
+Here we are stating that the Cloudformation stacks may be deployed to `us-east-2` only, while the Serverless
+application may be deployed to three US regions.
+
+When there are multiple deployments, they are created in the order they are listed in `runway.yml`
+(or destroyed in reverse order).  Runway does not attempt to determine the order itself based on
+dependencies between the deployments.  **((VERIFY!!))**
```
Comment (rgitzel) [inline docs/source/runway_config.rst]: True?

**Comment by troyready** [comment]
At a first glance, seems like it's moving in the right direction.

**Comment by troyready** [comment]
Closing for now; may still pull in some of this content in a future update

**Pull request closed without merging**